app/serializers/allocation_serializer.rb

```diff
@@ -19,5 +19,6 @@ class AllocationSerializer < ActiveModel::Serializer
   has_one :to_location
   has_many :moves
 
+  MINIMUM_RELATIONSHIPS = { from_location: [], to_location: [], moves: %i[person] }.freeze
```

Comment:
```suggestion
  MINIMUM_RELATIONSHIPS = %i[from_location to_location moves.person].freeze
```

Reply:
@willfish I'll add that in a later PR, was a bit too hasty on the merge button this time :)